README.adoc

```diff
@@ -1,45 +1,58 @@
-# tutorial-web-app
+= Tutorial Web App
+
+:toc:
```

Comment:
Put up at the very top @camilamacedo86

Reply:
Above the = Tutorial Web App

Reply:
Or if you want the toc to appear under the first heading then:

At the very top do:
:toc:
:toc-placement!:

and then where you want the TOC to appear do:
toc::[]